config/ols.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ols-proxy
+  name: ols-proxy
+spec:
+  ports:
+  - name: https
+    port: 8443
+    targetPort: https
+  selector:
+    app: ols-proxy
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ols-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ols-proxy
+  template:
+    metadata:
+      labels:
+        app: ols-proxy
+    spec:
+      securityContext:
+        runAsUser: 65532
+      serviceAccountName: ols-proxy
+      containers:
+      - name: ols-proxy
+        image: quay.io/brancz/kube-rbac-proxy:v0.15.0
```

Comment:
we should pull the openshift version of this image.  `quay.io/openshift/origin-kube-rbac-proxy:4.14.0` should work.

Reply:
Changed as suggested.